orchestration/dags/dependencies/import_analytics/import_analytics.py

```diff
@@ -376,7 +393,11 @@ def define_import_tables():
         "destination_dataset": "{{ bigquery_analytics_dataset }}",
         "destination_table": "firebase_aggregated_similar_offer_events",
         "time_partitioning": {"field": "event_date"},
-        "depends": ["diversification_booking", "enriched_user_data"],
+        "depends": [
+            "diversification_booking",
+            "enriched_user_data",
+            "similar_offer_events",
```

Comment:
remplacer par _analytics_firebase_similar_offer_events_  car la table similar_offer_events est identifiée par analytics_firebase_similar_offer_events dans le dictionnaire de config

Reply:
done in next commit